compliance/tests/struct_type_assertion/struct_type_assertion.gs.ts

```diff
@@ -7,14 +7,14 @@ export function main(): void {
 	let i: null | any = null
 	i = {Name: "Alice"}
 
-	let { value: s, ok: ok } = $.typeAssert<any/* unhandled type: *types.Struct */>(i, {kind: $.TypeKind.Struct, fields: new Set(['Name'])})
+	let { value: s, ok: ok } = $.typeAssert<{}>(i, {kind: $.TypeKind.Struct, fields: new Set(['Name']), methods: new Set()})
```

Comment:
I don't think the type assertion should be {} here.

Reply:
Delete the skip-test and skip-... Files from function_type_assertion compliance test